tests/main/interfaces-many/task.yaml

```diff
@@ -13,9 +13,10 @@ details: |
 # - Ubuntu classic 16.04 amd64 VM
 # - Ubuntu classic 18.04 amd64 VM
 # - All Ubuntu autopkgtests
-# - Debian unstable amd64 VM
+# - Debian sid amd64 VM
+# - Debian 9 amd64 VM
 # - TODO: All Fedora systems (for classic-only; unrelated error elsewhere)
-systems: [ubuntu-core-16-64, ubuntu-14.04-32, ubuntu-16.04-64, ubuntu-18.04-64, ubuntu-*-amd64, ubuntu-*-armhf, ubuntu-*-arm64, ubuntu-*-i386, ubuntu-*-ppc64el, debian-unstable-64]
+systems: [ubuntu-core-16-64, ubuntu-14.04-32, ubuntu-16.04-64, ubuntu-18.04-64, ubuntu-*-amd64, ubuntu-*-armhf, ubuntu-*-arm64, ubuntu-*-i386, ubuntu-*-ppc64el, debian-sid-64, debian-9-64]
```

Comment:
This can be converted to a list:
```
systems:
  - ubuntu-core-16-64
  - ubuntu-14.04-32
   ...
```

Reply:
Wwe could use `debian-*` here instead of listing both explicitly.